src/components/Profile/ProfileInfo/ProfileDescriptionForm.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from "react";
+import {
+  createField,
+  Input,
+  TextArea,
+} from "../../common/FormsControls/FormsControls";
+import { reduxForm } from "redux-form";
+
+const ProfileDescriptionForm = ({handleSubmit, profile }) => {
+  return (
+    <form onSubmit={handleSubmit}>
+      <div>
+        <button>save</button>
+      </div>
+      <div>
+        <b>Full Name:</b> {createField("Full name", "fullName", [], Input)}
+      </div>
+      <div>
+        <b>Looking for a job:</b>
+        {createField("", "lookingForAJob", [], Input, { type: "checkbox" })}
+      </div>{" "}
+      <div>
+        {" "}
+        <b>My professional skills:</b>
+        {createField(
+          "JobDescription",
+          "lookingForAJobDescription",
+          [],
+          TextArea
+        )}
+      </div>
+      <div>
+        <b>About me:</b> {createField("aboutMe", "aboutMe", [], TextArea)}
+      </div>
+      <div>
+        <b>Contacts:</b>{" "}
+        {Object.keys(profile.contacts).map((key) => {
```

Comment:
Consider breaking this line down into several functions/variables.
E.g. `Object.keys(...).map( () => {...})` could be
`const arr = Object.keys(...);`
`const mapHandler = () => {...};`
`const result = arr.map(mapHandler);`

Reply:
done